geth/node/status_node.go

```diff
@@ -121,6 +126,19 @@ func (n *StatusNode) start(services []node.ServiceConstructor) error {
 	return n.gethNode.Start()
 }
 
+func (n *StatusNode) setupWhisper() error {
```

Comment:
It's named `setupWhisper` but it creates a time manager and updates time source in Whisper. 

It would be nice to do it when we register Whisper (`activateShhService()`) but it does not look like easy. 

Can we maybe call this method `setWhisperTimeSource()` to be more explicit?